backend/src/controllers/user.controller.ts

```diff
@@ -24,4 +25,10 @@ userController.get('/', verifyToken, // you can add middleware on specific reque
     }
 );
 
+userController.post('/delete',
+    (req: Request, res: Response) => {
+        adminService.delete(req.body).then(num => res.send(num)).catch(err => res.status(500).send(err));;
```

Comment:
Here you could use directly:
`userController.delete('/:id'...`
also you should verify the token for each such method, otherwise a not logged in user could delete a user